packages/jest-diff/README.md

```diff
@@ -240,7 +242,7 @@ const options = {
 
 The `jest-diff` package does not assume that the 2 labels have equal length.
 
-### Example of options for colors
+### Example of options for colors of change lines
```

Comment:
```suggestion
### Example of options for colors of changed lines
```
?

"change line" reads weird to me, but I'm not a native speaker so feel free to ignore 🙂

Reply:
Yes, thank you. I pushed a commit with changes to 2 occurrences each:

* s/change line/changed line/
* s/change substring/changed substring/